### PR TITLE
Migrate mirrors to .org

### PR DESCRIPTION
**Pull request by the-maldridge**
This chain should change the default provided mirror from .eu to .org for all packages that reference it, and should change travis as well.

### CODE REVIEW

**Comment by the-maldridge** [comment]
Build fails due to rust.

**Comment by maxice8** [comment]
Please rebase since `xbps` has been revbumped

**Comment by Gottox** [comment]
Looks good. I have a look into rust.

**Comment by Gottox** [comment]
But I expect that it's just an issue on travis

**Comment by the-maldridge** [comment]
Since its only data that affects future builds of rust, not anything that gets baked in, we could just forego the revbump.

**Comment by the-maldridge** [comment]
I un-revbumped things that won't actually change, since this just changes the address the distfiles that are fetched from, not the files themselves.

File: common/travis/set_mirror.sh [hunk 1]
```diff
@@ -11,5 +11,5 @@ for _i in etc/repos-remote.conf etc/defaults.conf etc/repos-remote-x86_64.conf ;
     sed -i "s:https:$TRAVIS_PROTO:g" $_i
 
     # Now set the mirror
-    sed -i "s:repo\.voidlinux\.eu:$TRAVIS_MIRROR:g" $_i
+    sed -i "s:alpha\.de\.repo\.voidlinux\.org:$TRAVIS_MIRROR:g" $_i
```
Comment (fosslinux) [inline common/travis/set_mirror.sh]: shouldn't this be consistent with `$TRAVIS_MIRROR`?

File: etc/defaults.conf [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # [OPTIONAL]
 # Enable optional arguments to xbps-install for the host system.
 # Currently used in the binary-bootstrap bootstrap-update targets.
-XBPS_INSTALL_ARGS="--repository=https://repo.voidlinux.eu/current --repository=https://repo.voidlinux.eu/current/musl --repository=https://repo.voidlinux.eu/current/aarch64"
+XBPS_INSTALL_ARGS="--repository=https://alpha.de.repo.voidlinux.org/current --repository=https://alpha.de.repo.voidlinux.org/current/musl --repository=https://alpha.de.repo.voidlinux.org/current/aarch64"
```
Comment (fosslinux) [inline etc/defaults.conf]: same here (`$TRAVIS_MIRROR`)

File: etc/repos-remote-x86_64.conf [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Remote repositories
-repository=https://repo.voidlinux.eu/current/multilib
-repository=https://repo.voidlinux.eu/current/multilib/nonfree
+repository=https://alpha.de.repo.voidlinux.org/current/multilib
```
Comment (fosslinux) [inline etc/repos-remote-x86_64.conf]: same here

Comment (fosslinux) [reply etc/repos-remote-x86_64.conf]: and so forth down

File: common/travis/set_mirror.sh (hunk 1 above)
Comment (the-maldridge) [reply common/travis/set_mirror.sh]: Uhhh no.  This entire block sets the repos to whatever $TRAVIS_MIRROR contains.

**Comment by Gottox** [comment]
Looks good to me :+1:

**Pull request merged**